src/client/Microsoft.Identity.Client/Platforms/iOS/AuthenticationContinuationHelper.cs

```diff
@@ -24,14 +24,33 @@ public static bool SetAuthenticationContinuationEventArgs(NSUrl url)
         }
 
         /// <summary>
-        /// Returns if the response is from the broker app
+        /// Returns if the response is from the broker app. See https://aka.ms/msal-net-ios-13-broker
+        /// for more details.
         /// </summary>
         /// <param name="sourceApplication">application bundle id of the broker</param>
         /// <returns>True if the response is from broker, False otherwise.</returns>
         public static bool IsBrokerResponse(string sourceApplication)
         {
-            Debug.WriteLine("IsBrokerResponse Called with sourceApplication {0}", sourceApplication);
-            return sourceApplication != null && sourceApplication.Equals("com.microsoft.azureauthenticator", StringComparison.OrdinalIgnoreCase);
+            Debug.WriteLine("IsBrokerResponse called with sourceApplication {0}", sourceApplication);
+
+            if (sourceApplication != null && sourceApplication.Equals("com.microsoft.azureauthenticator", StringComparison.OrdinalIgnoreCase))
+            {
+                return true;
+            }
+            else if (string.IsNullOrEmpty(sourceApplication))
```

Comment:
if you don't need `else` becasue you return. Code is clearer like: 

`if (...) { return true} if (...) { return false } if (..) return x`